includes/library/zencart/platform/src/Paginator/adapters/QueryFactory.php

```diff
@@ -17,15 +17,16 @@
  */
 class QueryFactory extends AbstractAdapter implements AdapterInterface
 {
+
     /**
-     * getResultList method
-     *
-     * @return array|mixed
+     * @param $data
+     * @param array $params
+     * @return array
      */
-    public function getResultList($data, $params)
+    public function getResultList($data, array $params)
     {
         $limit = $params['currentItem'] - 1 . ',' . $params['itemsPerPage'];
-        $results = $data['dbConn']->execute($data['sqlQueries']['main'], $limit);
+        $results = $data['dbConn']->execute($data['mainSql'], $limit);
```

Comment:
I guess I should have said more about this in the commit comment. This version of the paginator classes are not currently used anywhere (the refactored listing box code will eventually use them). So the legacy zcPaginator classes are still left in code. Those files will be deleted eventually.